src/settings.service.js

```diff
@@ -368,12 +374,25 @@ plugin.service('wgnConfigSettings', ['$q', 'wgnConfigInputs', function ($q, conf
 		/**
 		 * Returns highlighted inputs.
 		 *
-		 * @return {Array<string>} An array of input ids.
+		 * @return {Array<string>} An array of input objects.
```

Comment:
@AParks I can't remember at this point but if it's really objects please update the docblock as well `{Array<Object>}`